copter/source/docs/terrain-following.rst

```diff
@@ -65,14 +65,14 @@ One common problem reported by users is the vehicle immediately disarms when the
 Terrain Spacing and Accuracy
 ============================
 
-The :ref:`TERRAIN_SPACING <TERRAIN_SPACING>` parameter controls the size of the grid used when requesting terrain altitude from the Ground Station (it is not used if using a Lidar). This is 100m by default but reducing to 30 may provide better accuracy at the expense of more telemetry traffic between the GCS and autopilot.  It is recommended that you use a :ref:`TERRAIN_SPACING <TERRAIN_SPACING>` of at least 30 meters.
+The :ref:`TERRAIN_SPACING <TERRAIN_SPACING>` parameter controls the size of the grid used when requesting terrain altitude from the Ground Station (it is not used if using a Lidar). This is 100m by default but reducing to 30 may provide better accuracy at the expense of more telemetry traffic between the GCS and autopilot, and 9x more file storage space on the SD card, but only the ground station uses a server with that resolution. MavProxy and Mission Planner currently do not. Also, if the vehicle is moving very fast, the autopilot may not be able to retrieve and cache the data quickly enough for the increased resolution to be actually used.  It is therefore recommended that you use a :ref:`TERRAIN_SPACING <TERRAIN_SPACING>` of at least 100 meters.
```

Comment:
"but only the ground station.." should probably be "but only **if** the ground station.."

Reply:
Maybe we can remove the "at least" from the "It is therefore recommended that you use a :ref:`TERRAIN_SPACING <TERRAIN_SPACING>` of at least 100 meters" line

Reply:
thanks fixed